<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookDialog</class>
 <widget class="QDialog" name="BookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#borrowInput, #pushButton{
 	background-color: #4CAF50; 
 	border: none;
  	text-align: center;
  	text-decoration: none;
	font-weight: bold;
	color: white;
}

#BookDialog{
	border-image: url(:/images/assets/dialog_BG.jpg);
}

#removeBtn{
 	background-color: rgb(226, 32, 36); 
 	border: none;
  	color: white;
  	text-align: center;
  	text-decoration: none;
	font-weight: bold;
}

QLabel{
	font-weight: bold;
}
</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeBtn">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QTableView" name="personTable">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>681</width>
     <height>251</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="personInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="borrowInput">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Borrow</string>
   </property>
  </widget>
  <widget class="QLabel" name="bookAvaiable">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>420</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>561</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name:     </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="bookName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Category:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="bookCategory">
        <item>
         <property name="text">
          <string>Action</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Education</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Computers</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Programming</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Adventure</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Comedy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Crime</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Drama</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fantasy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Historical</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Historical fiction</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Horror</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Magical realism</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mystery</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Paranoid fiction</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Philosophical</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Political</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Romance</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Saga</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Satire</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Science fiction</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Social</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Speculative</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Thriller</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Urban</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Western</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Author:   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="bookAuthor"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Language:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="bookLanguage">
        <item>
         <property name="text">
          <string>German</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Arabic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>French</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>English</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Russian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Spanish</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Chinese</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vietnamese</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Quantity:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="bookQuantity"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Format</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="bookFormat">
        <item>
         <property name="text">
          <string>PDF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PAPER</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EPUB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MOBI</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TXT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DOC/DOCX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DJVU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>KINDLE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PRC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HTML/XML</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Page:      </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="bookPage"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="hasDigital">
      <property name="text">
       <string>has Digital</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>ISBN:      </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="bookISBN"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
